src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -38,15 +40,29 @@ public class FindCommand extends Command {
 
```

Comment:
I think you forgot to change `MESSAGE_USAGE` here, I think it would be better to specify that the find command now supports `chained` preamble here